src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/NotYetReady/IfError.txt

```diff
@@ -0,0 +1,177 @@
+
+// ********** NON-ERROR PARAMETERS **********
+
+>> IfError(1, 2)
+1
+
+>> IfError(1, 2, 3)
+3
+
+>> IfError(1, 2, 3, 4)
+3
+
+>> IfError( 10, 11, 20, 21, 300 )
+300
+
+>> IfError(1, "error 1", 2, "error 2", "no error")
+"no error"
+
+>> IfError({a:1, b:"hello"}, {a:2, b:"good morning"}).b
+"hello"
+
+>> IfError(1, {a:1}, 2, {a:2}, {a:3}).a
+3
+
+>> IfError({a:{a1:true},b:"hello"}, {a:1,b:"error 1"}, {a:{a1:{a2:3}},b:"world"}, {a:1,b:"error 2"}, {a:1,b:"valid"}, {a:1,b:"error 3"}).a
+1
+
+
+// ********** ERROR PARAMETERS *************
+
+>> IfError(1/0, 3)
+3
+
+>> IfError(1/0, -1)
+3
+
+>> IfError(1/0, FirstError.Kind)
+13
+
+>> IfError(1/0, Error({Kind:FirstError.Kind}))
+#Error(Kind=Div0)
+
+>> IfError(Error({Kind:12}), FirstError.Kind)
+12
+
+>> IfError(0, 1, Sqrt(-2), 2)
+2
+
+>> IfError(0, 1, Error({Kind: ErrorKind.Unknown}), 2)
+2
+
+>> IfError(1/0, "error 1", 2, "error 2", "no error")
+"error 1"
+
+>> IfError({a:1, b:"hello"}, {a:2, b:"good morning"}, {a:1/0, b:"Error"}).b
+"Error"
+
+>> IfError({a:1, b:"hello"}, {a:2, b:"good morning"}, {a:1/0, b:"Error"}).a
+#Error(Kind=Div0)
+
+>> IfError( 1/0, Notify( "There was an internal problem" ) )
+1
+
+>> IfError(Error({Kind: ErrorKind.Validation}),"Error")
+"Error"
+
+>> IfError(Error({Kind: 11}),"Error")
+"Error"
+
+>> IfError(Error({Kind:ErrorKind.InvalidArgument}),"Error")
+#Error(Kind=InvalidArgument)

```

Comment:
This looks wrong - it should be the same result of the test with ErrorKind.Validation

Reply:
Corrected the expected result to "Error"